src/hotspot/cpu/x86/x86.ad

```diff
@@ -3879,6 +3880,18 @@ instruct roundD_reg(legRegD dst, legRegD src, immU8 rmode) %{
   ins_pipe(pipe_slow);
 %}
 
+instruct roundD_reg_avx(legRegD dst, legRegD src, immU8 rmode) %{
+  predicate(UseAVX > 0);
+  match(Set dst (RoundDoubleMode src rmode));
+  format %{ "roundsd $dst,$src" %}
+  ins_cost(150);
+  ins_encode %{
+    __ vroundsd($dst$$XMMRegister, $src$$XMMRegister, $src$$XMMRegister, $rmode$$constant);
+  %}
+  ins_pipe(pipe_slow);
+%}
+
+/*
 instruct roundD_mem(legRegD dst, memory src, immU8 rmode) %{
```

Comment:
The roundD_mem instruct could be removed now that it is not used. Also the PR could be titled as "Improve performance of Math ceil,  floor, and rint for x86".

Reply:
OK, let's remove roundD_mem to avoid confusion.